README.md

```diff
@@ -1,393 +1,296 @@
+```js
+const testee = require('testee');
 
+const browsers = 'phantom';
+const config = { reporter: 'spec' };
+const files = ['test/test.html'];
 
-### Capturing `console.log` and `console.error`
+testee.test(files, browsers, config)
+  .then(() => console.log('done!'));
+```
 
-Testee tracks all calls to `console.log` and `console.error` in a test. To get the output during a test run, set the `DEBUG` environment variable to `testee:console-log`:
 
-> `DEBUG=testee:console-log testee --browsers canary tests/jasmine.html`
+## Grunt Usage
 
-### Debugging
+This package comes with a [grunt](https://npmjs.com/grunt) task that extends the [configuration API](#configuration-api):
 
-Testee uses the Node [debug](https://github.com/visionmedia/debug) module. Detailed debugging information can be enabled
-in any environment (command line, Grunt, programatically) by setting the `DEBUG` environment variable to `testee:*`:
+```js
+module.exports = function(grunt) {
+  grunt.initConfig({
+    testee: {
+      options: {
+        // See configuration API
+      },
+      coverage: {
+        options: {
+          // See configuration API
+        },
+        src: ['test/index.html']
+      },
+      browserstack: {
+        options: {
+          // See configuration API
+        },
+        src: ['test/index.html']
+      }
+    }
+  });
 
-> `DEBUG=testee:* testee --browsers canary tests/jasmine.html`
+  grunt.loadNpmTasks('testee');
+};
+```
 
 
 ## Configuration API
 
-The following sections describe the available options for
-
-* the JSON configuration file if you run Testee from the command line using the `-c` or `--config` option
-* the [GruntJS task](#gruntjs) task configuration
-* programmatic usage with NodeJS
-
-
-### Default configuration
-
-Any options will be merged with the following default configuration:
-
-```js
+A simple, local config (JSON) example:
+```json
 {
-  port: 3996,
-  root: process.cwd(),
-  reporter: 'Dot',
-  timeout: 120,
-  delay: 1000,
-  tunnel: {
-    type: 'local'
+  "port": 3996,
+  "root": "/var/www/app/",
+  "reporter": "dot",
+  "timeout": 120,
+  "delay": 1000,
+  "tunnel": {
+    "type": "local"
   },
-  launch: {
-    type: 'local'
-  }
+  "launch": {
+    "type": "local"
+  },
+  "browsers": ["chrome", "firefox"]
 }
 ```
 
+An advanced, BrowserStack config (JS) example:
+```js
+require('dotenv').config();
 
-### General settings
-
-__`root` *{String}*__<br />
-Every time when running a test, Testee will start a static file server, by default in the current folder.
-That way, any test HTML file you reference will be loaded properly. The `root` option allows you to change
-the root path of the static fileserver.
-
-__`port` *{Integer}*__<br />
-The port for the static file server to start on. This will also be used by [Localhost tunneling services](#localhost_tunneling). The default is `3996`.
+const {name:project, version:build} = require('./package.json');
```

Comment:
Same as above, but also this will cause horizontal scrolling in the `browser` property.